src/sandstorm/grain.capnp

```diff
@@ -633,7 +633,20 @@ interface SessionContext {
   #   saveLabel: A string petname to give this label. This will be displayed to the user as the name
   #          for this capability.
   #
-  # (eg. window.parent.postMessage({powerboxRequest: {rpcId: myRpcId, query: [{}]}}, "*")
+  # e.g.:
+  #   window.parent.postMessage({
+  #     powerboxRequest: {
+  #       rpcId: myRpcId,
+  #       query: [
+  #         {
+  #           tags: [
+  #             { id: "10145426045091321204" },
```

Comment:
So, making this change has unveiled a second problem, which is that to do tag id matching, the `uint64_t`'s representation needs to be canonicalized, and it's awkward to canonicalize bignum-representing-strings in the browser without a bignum library or something like that.

My options now appear to be:

* require base-10 strings everywhere, since that's what node-capnp will produce by default when I refer to `Grain.UiView.typeId` on the server.  Requires apps to use the base 10 id representation.  What this patchset, as-is, implements.
* require base-16 strings everywhere, since that's what `capnp compile -o capnp` gives you.  Requires manual conversion of base10 strings to base16 strings on the server.  Requires apps use base16 strings only.
* Require both apps and sandstorm to convert strings into uint8 arrays or something like that.  This makes the client-side comparison easy and correct, but means weird conversions that don't mix cleanly with the other APIs on either side.
* Implement/use a pure-JS bignum library in the browser to parse and compare the two string values.  This does not put requirements on what the app code sends.

None of these options seem particularly great.  Do you have a preference for any of these?

In all cases, we still need to make a decision on what the shell should `postMessage()` back to the app in the powerboxDescriptor when the user grants a token - should it be a base10 string, a base16 string, or something else?  Regardless of what we choose, the app's backend is going to have to parse it into a 64-bit uint.